### DISCUSSION TITLE
How can I add remark42 comments to my blog?

### DISCUSSION THREAD

**Discussion by harisharjun** [CaiJimmy/hugo-theme-stack#771] [Q&A]
What values should be added in the host and site fields?
I do not understand what server URL I have to use, if so? Please help.

**Comment by zhixuan2333** [comment]
Have u already deployed your server? If yes.
- REMARK_URL – the URL where is Remark42 instance is served, passed as to backend REMARK_URL
- YOUR_SITE_ID - the that you passed to Remark42 instance on start, by default.SITEremark

If no look this docs to deploy your server!
https://remark42.com/docs/getting-started/installation/

**Reply by harisharjun** [reply]
I simply hosted my website (https://arjunuvacha.com) using Firebase. Upon exploration, I found that the real-time database URL for it is [https://arjunuvacha-3de80-default-rtdb.firebaseio.com/]

Is this the server you are referring to? I am not sure how to set up a  remark42 instance on a firebase server!!

I am not from Computer Science, so, do not fully comprehend it 😞  Request that you can take a look at my website or my repo https://github.com/harisharjun/arjunuvacha and help me out here, please 🙏

**Reply by CaiJimmy** [reply]
> I am not sure how to set up a remark42 instance on a firebase server!!

It does not seem possible to do so since the installation requires command-line access to the server, according to the official installation guide.
